gui/gui.py

```diff
@@ -2,203 +2,203 @@
 import locale
 
 class Box:
-    def __init__(self, gui, window, x, y, w, h, color):
-        self.gui = gui
-        self.window = window
-        self.x = x
-        self.y = y
-        self.w = w
-        self.h = h
-        self.color = color
-
-    def draw(self):
-        self.gui.box(self.window, self.x, self.y, self.w, self.h, self.color)
+	def __init__(self, gui, window, x, y, w, h, color):
```

Comment:
nit: you appear to have reindented this file to an unusual non 4 spaced indent.  the only meaningful change appears to be s/xrange/range/.